design/deployment_design.md

```diff
@@ -0,0 +1,70 @@
+#  **Dockerless CLI -** deployment
+
+This document is the design document for the deployment of the Python CLI. 
+
+Currently, our Python CLI can be deployed in the following ways:
+
+1. Install the CLI via the pip Python package manager like so: `pip3 install conjur-client`. If installed in this way, Python **is** required on machine. If installed via pip, the user can then run the CLI like so `conjur-cli --help`
```

Comment:
```suggestion
1. Install the CLI via the pip Python package manager like so: `pip3 install conjur-client`. If installed in this way, **Python3 is required** on the machine if installing the CLI this way. If installed via pip, the user can then run the CLI like so `conjur-cli --help`
```

Reply:
sounds a bit weird because if installing in this way appears at the beginning and end of the sentence. I changed it to:

```
**Python is required** on machine if installing the CLI in this way
```
is this clearer?